sdk/identity/azure-identity/src/main/java/com/azure/identity/credential/AppServiceMSICredential.java

```diff
@@ -30,9 +30,12 @@
         Configuration configuration = ConfigurationManager.getConfiguration();
         if (configuration.contains(BaseConfigurations.MSI_ENDPOINT)) {
             msiEndpoint = configuration.get(BaseConfigurations.MSI_ENDPOINT);
-        }
+        } else
```

Comment:
We are missing an opening / closing brace here? CheckStyle should be failing here shouldn't it? Please make sure it does.